src/main/java/io/strimzi/kafka/bridge/http/HttpAdminClientEndpoint.java

```diff
@@ -156,7 +156,7 @@ public void doGetTopic(RoutingContext routingContext) {
                 root.put("partitions", partitionsArray);
                 HttpUtils.sendResponse(routingContext, HttpResponseStatus.OK.code(), BridgeContentType.KAFKA_JSON, root.toBuffer());
 
-            } else if (done.cause() instanceof UnknownTopicOrPartitionException) {
+            } else if (done.cause() != null && done.cause().getCause() instanceof UnknownTopicOrPartitionException) {
```

Comment:
done.cause() was returning `CompletionException` instead of the `UnknownTopicOrPartitionException` or you can also say that with the latest vertx and kafka, the `UnknownTopicOrPartitionException` comes wrapped inside the `CompletionException`. So I had to use done.cause().getCause() get the `UnknownTopicOrPartitionException` out of the done.cause() which was `CompletionException`